bin/check_pylint_diff

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+# Created By: miha@reciprocitylabs.com
+# Maintained By: miha@reciprocitylabs.com
+
+set -o nounset
+set -o errexit
+
+ARG1=${1:-}
+TMP_REPO=".temp_repo"
+SCRIPT=$(basename "$0")
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+
+cd "${SCRIPTPATH}/.."
+
+
+if [[ "$ARG1" == "-h" || "$ARG1" == "--help" ]]; then
+  echo "
+Usage: $SCRIPT [test_commit] [-h]
+
+This script will compare pylint error count from two different commits.
+Note: all changes that are not committed will be ignored.
+
+
+Given the commit tree:
+
+       D---E---F---G---H
+            \\         /
+             A---B---C
+
+- Running '$SCRIPT' on C will check the diff between B and C
```

Comment:
@kernc Okay, maybe the text is not clear, but it won't check last two commits, but only changes in the last commit. last commit is basically the diff between previous commit and current state.

As for why we don't check anything else:
* If I'm on a merge commit I know which two branches to compare.
* If I'm on a normal commit, I don't know anything about any other branch (unless that info is given as an argument)

Taking `gh:biolab/orange3 master" branch is useless for my fork. Trying to figure out which branch to take is really hard and prone to errors.